src/openpgp.js

```diff
@@ -73,7 +73,7 @@ export function generateKey({ userIDs = [], passphrase = "", type = "ecc", rsaBi
       revocationCertificate: revocationCertificate
 
     };
-  }).catch(onError.bind(null, 'Error generating keypair'));
+  }).catch(err => onError('Error generating keypair', err));
```

Comment:
I'd try to be consistent within this file, and personally I prefer the variant with the `try/catch`, so I'd use that in all these functions (and try to get rid of `.then` and `.catch`)